This is an awful hack. The test harness should not merge streams.
--- sudo-1.8.27/plugins/sudoers/regress/cvtsudoers/test32.sh
+++ sudo-1.8.27/plugins/sudoers/regress/cvtsudoers/test32.sh
@@ -4,7 +4,7 @@
 #
 
 exec 2>&1
-./cvtsudoers -c "" -b "ou=SUDOers,dc=my-domain,dc=com" -O 1000 -P 1 <<EOF
+stdbuf -o 1M ./cvtsudoers -c "" -b "ou=SUDOers,dc=my-domain,dc=com" -O 1000 -P 1 <<EOF
 user0  ALL = (ALL:ALL) ALL
 user1  ALL = (ALL:ALL) ALL
 user2  ALL = (ALL:ALL) ALL
